tests/test_adjoint_jacobian.py

```diff
@@ -867,10 +867,8 @@ def circuit(p):
                 if operation.num_params == 3:
                     qml.ctrl(
                         operation(*p, wires=range(n_qubits - num_wires, n_qubits)),
-                        control_wires,
-                        control_values=[
-                            control_value or bool(i % 2) for i, _ in enumerate(control_wires)
-                        ],
+                        control_wires[0],
```

Comment:
We could also 
```
if isinstance(operation, qml.Rot) and n_controls > 1:
    continue
```
above. This would prevent inadvertently skipping tests for future operations with 3 parameters supporting multiple controls.

Reply:
This seems more hacky to me than the current approach :) 

The current approach still checks `qml.Rot` gates with single control_wires.